scripts/dts/gen_defines.py

```diff
@@ -67,11 +72,10 @@ def main():
         #define DT_COMPAT_<COMPAT> 1
         out(f"COMPAT_{str2ident(compat)}", 1)
 
-    # Derived from /chosen
+    # Definitions derived from /chosen nodes.
```

Comment:
I skip the period when it's just one sentence, so there's a method to it at least.